### PR TITLE
Aman- Dashboard Layout 

### PR DESCRIPTION
**Pull request by shipglobalaman**

### CODE REVIEW

File: aman/index.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="overflow-hidden h-full">
```
Comment (shubhamships) [inline aman/index.html:2]: why this?

File: aman/index.html [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite + React</title>
   </head>
-  <body>
+  <body class="h-full overflow-auto">
     <div id="root"></div>
```
Comment (shubhamships) [inline aman/index.html:10]: we never add styles outside the parent root div

**Comment by shubhamships** [comment]
Everything else looks good to merge. @shipglobalaman please resolve these two comments and get it merged ASAP

File: aman/index.html (hunk 1 above)
Comment (shipglobalaman) [reply aman/index.html:2]: Sir, this is used to manage the scrollbar behavior by restricting it to the body element.

**Pull request merged**